babel.config.js

```diff
@@ -17,16 +17,8 @@ const defaultPlugins = [
 ];
 
 const webpack = {
-    env: {
-        production: {
-            presets: defaultPresets,
-            plugins: [...defaultPlugins, 'transform-remove-console'],
-        },
-        development: {
-            presets: defaultPresets,
-            plugins: defaultPlugins,
-        },
-    },
+    presets: defaultPresets,
+    plugins: defaultPlugins,
```

Comment:
- The `env.production` path were never engaged for web builds. The reason being, `NODE_ENV` isn't defined before initiating the build process.
- I'm omitting the 'transform-remove-console' plugin. Our intent is to retain console logs for web/desktop in production builds.

In essence, this modification aligns the dev and prod configurations, making them the same, hence the presented code change.